app/views/shared/_document.html.erb

```diff
@@ -2,9 +2,23 @@
   <div class="aspect-ratio-4x3">
     <div class="aspect-ratio-content">
       <img src="<%= document[:thumbnail_src] %>">
+      <% document.response[:highlighting][document.id][:text].tap do |highlights| %>
+        <% highlights.reject do |highlight|
+             term = highlight.match(/<em>([^<]*)<\/em>/)[1].downcase
+             document[:title].downcase.include?(term)
+             # If someone is doing a known item search,
+             # repeating the title just looks odd.
+        end.each do |highlight| %>
+          <div class="highlight">
+            &hellip;
+            <%= highlight.html_safe %>
+            &hellip;
+          </div>
+        <% end if highlights %>
+      <% end if document.class.method_defined? :response %>
```

Comment:
Good call on idioms.

So, in rendering the tabbed pages, we have things that look like solr results, but they're not:

https://github.com/WGBH/openvault3/blob/dac9d938d582f1e8fb8597ea0300fdf2bef61d01/app/models/tabbed.rb#L87

We have an array of `TabbedCell`, and it looks like it could be from RSolr, and that's been working fine for months.

Maybe we want code outside the view providing `document.response[:highlighting]`, and we could explicitly set it to be an empty hash, and that might be more clear...